httpclient/api_client.go

```diff
@@ -28,16 +27,25 @@ type ClientConfig struct {
 	AuthVisitor RequestVisitor
 	Visitors    []RequestVisitor
 
-	RetryTimeout       time.Duration
+	// The maximum amount of time to retry requests that return retriable errors.
+	// If unset, the default is 5 minutes.
+	RetryTimeout time.Duration
+
+	// Returns the amount of time to wait after the given attempt.
+	RetryBackoff retries.BackoffFunc
```

Comment:
This is added here and exposed initially for testing, though it should be safe for users to configure this if they need to modify the backoff behavior.